lib/api/controllers/documentController.js

```diff
@@ -729,6 +730,11 @@ class DocumentController extends NativeController {
 
     this.assertNotExceedMaxWrite(documents.length);
 
+    if (Object.prototype.hasOwnProperty.call(request.input.args, '_source') && 
```

Comment:
Use this to get this parameter and set it to `true` by default https://docs.kuzzle.io/core/2/framework/classes/kuzzle-request/get-string/

Reply:
Why not directly accessing the property with `request.input.args._source` ? Using `Object.prototype.hasOwnProperty.call` is very slow and not readable.

Either way you should use the `getBoolean` method to correctly retrieve boolean flags

Reply:
What is the issue of this discussion? Why has it been marked as resolved?

Reply:
Given the surrounding conversations, I frankly do not understand why this very simple bit of code is in this shape.